sync.yml

```diff
@@ -1,3 +1,22 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: scraper-sync-public-service
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '9090'
```

Comment:
I think there needs to be a correspondence between the port scraped by prometheus and the port config on the service spec below. Prometheus will try to contact the service address on this port (9090).

Is it possible to add multiple ports to the spec below?

Reply:
Done